src/components/select/select.scss

```diff
@@ -91,7 +91,11 @@ $mdc-select-bottom-line-hover-color: $lime-text-field-bottom-line-color;
             background-color: var(--mdc-theme-primary);
         }
 
-        .limel-select__dropdown-icon {
+        .mdc-select__dropdown-icon {
+            // width: pxToRem(28);
+            // margin-left: pxToRem(8);
+            // margin-top: pxToRem(32);
+            // margin-bottom: pxToRem(8);
```

Comment:
We should remove this commented code :slightly_smiling_face:

Reply:
Removed